src/utils/README.md

```diff
@@ -0,0 +1,8 @@
+Extra Information and Tips
```

Comment:
this is the wrong place for this file but I don't know where it /should/ go. I'll think about it.